Package.swift

```diff
@@ -165,8 +166,8 @@ var package = Package(
     ),
     .target(
       name: "BraveWidgetsModels",
-      dependencies: ["BraveShared"],
-      sources: ["BraveWidgets.intentdefinition", "LockScreenFavoriteIntentHandler.swift"],
+      dependencies: ["Favicon", "LoggerPlugin"],
```

Comment:
`LoggerPlugin` should be part of `plugins` not `dependencies`

Reply:
Fixed